README.md

```diff
@@ -25,117 +25,120 @@ The I2Cwrapper framework consists of **four basic components.** The first two dr
-* **ServoI2C**: Control servo motors via I2C. Works just like the plain Arduino servo library.
+* **AccelStepperI2C**: Control up to eight stepper motors with acceleration control via Mike McCauley's [AccelStepper library](https://www.airspayce.com/mikem/arduino/AccelStepper/index.html), and up to two end stops per stepper. Uses a state machine and an optional controller interrupt line to prevent I2C bus clogging. Works nearly like the original library. 
+* **ServoI2C**: Control servo motors via I2C. Works just like the plain Arduino [`Servo` library](https://www.arduino.cc/reference/en/libraries/servo).
 * **PinI2C**: Control the digital and analog in- and output pins of the target device via I2C, similar to an IO-expander. Works just like as the plain Arduino digitalRead() etc. commands.
 * **ESP32sensorsI2C**: Read an ESP32's touch sensors, hall sensor, and (if available) temperature sensor via I2C. Uses the optional controller interrupt line to inform the controller about a touch button press. Works nearly like the original ESP32 functions.
 
-Modules can be selected in **any combination** at compile time for a specific target (see below for details). It is easy to **[add new modules](#How to add new modules)** with help of the provided **templates**.
+Modules can be selected in **any combination** at compile time for a specific target (see below for details). It is easy to **[add new modules](#how-to-add-new-modules)** with help of the provided **[templates](templates)**.
 
 ## Limitations
 
-### Limitations for end users
+### Limitations for End Users
 
-   - Arduinos aren't perfect I2C target devices. Particularly, the Arduino's Wire library doesn't support [clock stretching](https://onlinedocs.microchip.com/pr/GUID-CA6B1F8F-E09D-4780-A52A-CBBF61902464-en-US-2/GUID-5CCAB0DB-28BD-4095-B2E2-2F3CF0FC6966.html) in a way that allows the target to finish reacting to the previous command if it hasn't done so yet before the transmission occurred. That's why it's important to make sure that the **target is not flooded** with commands or requests with too little time to handle them. I2Cwrapper provides an [adjustable minimum delay](# Adjusting the I2C delay) between transmissions to handle that problem.
+   - Arduinos aren't perfect I2C target devices. Particularly, the Arduino's Wire library doesn't support [clock stretching](https://onlinedocs.microchip.com/pr/GUID-CA6B1F8F-E09D-4780-A52A-CBBF61902464-en-US-2/GUID-5CCAB0DB-28BD-4095-B2E2-2F3CF0FC6966.html) in a way that allows the target to finish reacting to the previous command if it hasn't done so yet before the transmission occurred. That's why it's important to make sure that the **target is not flooded** with commands or requests with too little time to handle them. I2Cwrapper provides an [adjustable minimum delay](#adjusting-the-i2c-delay) between transmissions to handle that problem (and diagnostics to determine the delay).
```

Comment:
I'm not sure about the diagnostics functions. They are a relict from AccelStepperI2C,  are not written with the module approach in mind and need a revision. That's why I chose to omit them here for the moment.

Reply:
removed text